src/test/lib/TestHelpers.h

```diff
@@ -581,6 +581,26 @@ class DatapathHooks
     }
 };
 
+struct EcnModifyHelper : public DatapathHook
+{
+    CXPLAT_ECN_TYPE EcnType;
```

Comment:
Default?

Reply:
Oh right, it actually needs a IsSet flag that is default FALSE.

Reply:
fixed